Library/Formula/coreutils.rb

```diff
@@ -29,6 +29,7 @@ class Coreutils < Formula
     depends_on "texinfo" => :build
     depends_on "xz" => :build
     depends_on "wget" => :build
+    depends_on "gperf" => :build
```

Comment:
Please change this line to
``ruby
depends_on "homebrew/dupes/gperf" => :build unless OS.mac?
```